lib/charms/ip_router_interface/v0/ip_router_interface.py

```diff
@@ -449,6 +451,46 @@ def request_network(self, networks: List[Network], custom_network_name: str = No
             str([r.name for r in ip_router_relations]),
         )
 
+    def get_routing_table(self) -> RoutingTable:
+        """Fetches combined routing tables made available by ip-router providers
+
+        Returns:
+            A list of objects of type `Network`. This list contains networks
```

Comment:
The type here does not match the return typing

Reply:
Fixed